ci: cannot find module 'nan'

The following error occurs when running `npm i` on `macos-latest` ([example](https://github.com/robingenz/dhbw-dualis-app/runs/1482723411?check_suite_focus=true)):

```
> fsevents@1.2.13 install /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/webpack-dev-server/node_modules/fsevents
> node install.js

internal/modules/cjs/loader.js:834
  throw err;
  ^

Error: Cannot find module 'nan'
Require stack:
- /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/webpack-dev-server/node_modules/fsevents/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:120:18)
    at Object.runInThisContext (vm.js:309:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at evalScript (internal/process/execution.js:94:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/webpack-dev-server/node_modules/fsevents/[eval]'
  ]
}
gyp: Call to 'node -e "require('nan')"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/runner/hostedtoolcache/node/12.19.0/x64/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Darwin 19.6.0
gyp ERR! command "/Users/runner/hostedtoolcache/node/12.19.0/x64/bin/node" "/Users/runner/hostedtoolcache/node/12.19.0/x64/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/webpack-dev-server/node_modules/fsevents
gyp ERR! node -v v12.19.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 

> fsevents@1.2.13 install /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/watchpack-chokidar2/node_modules/fsevents
> node install.js

internal/modules/cjs/loader.js:834
  throw err;
  ^

Error: Cannot find module 'nan'
Require stack:
- /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/watchpack-chokidar2/node_modules/fsevents/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:120:18)
    at Object.runInThisContext (vm.js:309:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at evalScript (internal/process/execution.js:94:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/watchpack-chokidar2/node_modules/fsevents/[eval]'
  ]
}
gyp: Call to 'node -e "require('nan')"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/runner/hostedtoolcache/node/12.19.0/x64/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Darwin 19.6.0
gyp ERR! command "/Users/runner/hostedtoolcache/node/12.19.0/x64/bin/node" "/Users/runner/hostedtoolcache/node/12.19.0/x64/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/watchpack-chokidar2/node_modules/fsevents
gyp ERR! node -v v12.19.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 

> core-js@3.6.5 postinstall /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> puppeteer@5.5.0 install /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/puppeteer
> node install.js


Chromium (818858) downloaded to /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/puppeteer/.local-chromium/mac-818858

> @angular/cli@11.0.2 postinstall /Users/runner/work/dhbw-dualis-app/dhbw-dualis-app/node_modules/@angular/cli
> node ./bin/postinstall/script.js

added 1857 packages in 46.4s

```

--- a/package.json
+++ b/package.json
@@ -57,6 +57,7 @@
     "karma-coverage-istanbul-reporter": "3.0.3",
     "karma-jasmine": "4.0.1",
     "karma-jasmine-html-reporter": "1.5.4",
+    "nan": "2.14.2",
     "prettier": "2.2.1",
     "protractor": "7.0.0",
     "puppeteer": "5.5.0",

